File: Module 3/Mastering Swift 2_Code/B05115_13_codes/Projects/SwiftProject/SwiftProject/MessageBuilder.m
//
//  MessageBuilder.m
//  SwiftProject
//
//  Created by Jon Hoffman on 2/12/15.
//  Copyright (c) 2015 Jon Hoffman. All rights reserved.
//

#import "MessageBuilder.h"
#import "SwiftProject-Swift.h"

@implementation MessageBuilder

-(NSString *)getPersonalizedMessageForName:(NSString*)name {
    Messages *messages = [[Messages alloc] init];

    NSString *retString = [NSString stringWithFormat:@"To: %@, %@",name, [messages getMessage]];
    return retString;
}

@end

File: Module 3/Mastering Swift 2_Code/B05115_13_codes/Projects/ObjectiveCProject/ObjectiveCProject/MessageBuilder.swift
//
//  MessageBuilder2.swift
//  ObjectiveCProject
//
//  Created by Jon Hoffman on 2/12/15.
//  Copyright (c) 2015 Jon Hoffman. All rights reserved.
//

import Foundation

class MessageBuilder: NSObject {
    
    func getPersonalizedMessage(name: String) -> String {
        let messages = Messages()
        let retMessage = "To: " + name + ", " + messages.getMessage()
        
        return retMessage;
    }
}